pallets/ctype/Cargo.toml

```diff
@@ -12,41 +12,59 @@ targets = ["x86_64-unknown-linux-gnu"]
 [build-dependencies.substrate-wasm-builder-runner]
```

Comment:
The dependencies have just, for some reason, been ordered alphabetically, nothing more.

Reply:
I like the sorting! 👍🏻